lib/perl/Genome/Config.t

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env genome-perl
+
+use strict;
+use warnings;
+
+use Genome::Test::Config qw(setup_config);
+
+use Test::More tests => 6;
+use Test::Fatal qw(exception);
+
+
+subtest 'Genome does not need to be loaded for Genome::Config' => sub {
+    use_ok('Genome::Config');
+    ok(!scalar(grep { $_ eq 'Genome.pm' } keys %INC), 'Genome has not been loaded');
+};
+
+subtest 'basic lookup' => sub {
+    plan tests => 2;
+
+    my ($temp_dirs, $new_temp_dir) = Genome::Test::Config::temp_dir_helper();
```

Comment:
I believe that, as long as the caller here holds a reference to `$new_temp_dir`, the `@temp_dirs` inside will not be cleaned up even if a reference to that array is not held here.